### PR TITLE
add blog address only in Chinese

### PR DESCRIPTION
**Pull request by DavadDi**
add blog address only in Chinese

### CODE REVIEW

**Comment by lmb** [comment]
Can we get a netlify preview?

**Comment by dthaler** [comment]
This PR only adds the link on the Chinese home page, not the English or French ones.   Should it be listed on all three for consistency?   It looks ok when I use automatic translation to view it in English.

**Comment by DavadDi** [comment]
> This PR only adds the link on the Chinese home page, not the English or French ones. Should it be listed on all three for consistency? It looks ok when I use automatic translation to view it in English.

This sounds like a very good suggestion, and I will add the relevant links in English and French.

File: src/pages/index.fr-fr.js [hunk 1]
```diff
@@ -139,6 +139,9 @@ const Outro = () => (
               <li>
                 <a href="/projects">Liste de projets basés sur eBPF</a>
               </li>
+              <li>
+                <a href="https://www.ebpf.top">Tête première eBPF (Chinois)</a>
```
Comment (qmonnet) [inline src/pages/index.fr-fr.js:143]: Hi and thanks! This is not entirely correct in French. I don't read Chinese, but a literal translation of the English title above could be `eBPF, la tête la première`; or `Plonger la tête la première dans eBPF` (“Dive head first into eBPF”), or just `Plongée dans eBPF` (“A dive into eBPF”).

**Review by qmonnet — CHANGES_REQUESTED** [review]

File: src/pages/index.fr-fr.js (hunk 1 above)
Comment (DavadDi) [reply src/pages/index.fr-fr.js]: Thanks a lot!

**Review by qmonnet — APPROVED** [review]

**Comment by xmulligan** [comment]
@DavadDi are you happy if this PR is merged or is there anything else you want to add?

**Comment by DavadDi** [comment]
@xmulligan Very happy, no more additions for now，thanks

**Pull request merged**